### PR TITLE
add link to forge

### PR DESCRIPTION
**Pull request by anton-iskryzhytskyi**
[Issue](https://github.com/rust-lang/www.rust-lang.org/issues/1101)

### CODE REVIEW

File: locales/en-US/common.ftl [hunk 1]
```diff
@@ -42,6 +42,7 @@ nav-logo-alt = Rust Logo
 footer-doc = Documentation
 footer-ask = Ask a Question on the Users Forum
 footer-status = Check Website Status
+footer-sup-doc = Supplementary Documentation
```
Comment (XAMPPRocky) [inline locales/en-US/common.ftl:45]: I feel like this is a bit more descriptive, as a lot of people know it as forge.
```suggestion
footer-sup-doc = Rust Forge (Contributor Documentation)
```

**Comment by anton-iskryzhytskyi** [comment]
> Thank you for your PR! Would you be able to remove the non-english translations? As localisation needs to go through pontoon for review. Once the feedback has been addressed I would be happy to merge it in.

Should I create new PRs for non-english translations? Russian is my native language, so I can translate it correctly in any case.

Working on correcting comments

Thank you for the fast response!

**Comment by XAMPPRocky** [comment]
If you'd like to help with the translation efforts you need to do through https://pontoon.rust-lang.org it's our translation software that we use to manage and coordinate the localisation teams, any changes made there are automatically synced daily. You can read more about how translating the website works in the document linked below.

https://github.com/rust-lang/www.rust-lang.org/blob/master/TRANSLATIONS.md

**Comment by XAMPPRocky** [comment]
Thank you again for your PR, and congrats on your first contribution! 🎉

**Pull request merged**